<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewSession</class>
 <widget class="QWidget" name="NewSession">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Session</string>
  </property>
  <widget class="QListWidget" name="setup_list_widget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>231</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="date_time">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>194</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="data_file">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="proc_save_bttn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Process and Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="select_data_bttn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>14</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date/Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>54</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Setup</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>254</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Data File</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
